TOSegmentedControl/TOSegmentedControl.m

```diff
@@ -1091,6 +1091,39 @@ - (BOOL)selectedSegmentReversed
     return segment.isReversed;
 }
 
+
+// -----------------------------------------------
+// Manually change thumb horizontal offset to connect it with UIPageController scroll
+
+- (void)setThumbHorizontalOffset:(CGFloat)offsetPercent direction:(NSInteger)direction
+{
+    //Get current thumb frame
+    CGRect frame = [self frameForSegmentAtIndex:self.selectedSegmentIndex];
```

Comment:
At some point duration the transition, the `selectedSegmentIndex` value will change. This might need to account for that.